conf/nginx.conf

```diff
@@ -20,6 +20,12 @@ location __PATH__/ {
         access_log off;
     }
 
+    # Build error access
+    error_page 404 /error/404.html;
+    location = __PATH__/error/404.html {
+        internal;
+    }
+
```

Comment:
This bit is already in the optional nginx-code-error.conf, shouldnt this block be removed ?